assets/src/edit-story/elements/index.js

```diff
@@ -73,14 +74,29 @@ export const createPage = (pageProps = null) => {
 };
 
 export const duplicatePage = (oldPage) => {
-  const { elements: oldElements, ...rest } = oldPage;
+  const { elements: oldElements, animations: oldAnimations, ...rest } = oldPage;
 
   // Ensure all existing elements get new ids
-  const elements = oldElements.map(({ type, ...attrs }) =>
-    createNewElement(type, attrs)
-  );
+  let elementIdTransferMap = {};
+  const elements = oldElements.map(({ type, ...attrs }) => {
+    const id = uuidv4();
+    elementIdTransferMap[attrs.id] = id;
+    return createNewElement(type, attrs, { id });
+  });
```

Comment:
Instead of this whole overrides thing, can't we do:

```suggestion
  const elements = oldElements.map(({ type, ...attrs }) => {
    const newElement = createNewElement(type, attrs, { id });
    elementIdTransferMap[attrs.id] = newElement.id;
    return newElement
  });
```

Reply:
ah yeaup. that works too and we wouldn't need to prevent the signature. Will update now, thanks!